eyepop/endpoint.py

```diff
@@ -122,6 +125,115 @@ async def connect(self, stop_jobs: bool = True):
                 response.raise_for_status()
             log_requests.debug('after PATCH %s', stop_jobs_url)
 
+    async def get_pop_comp(self) -> str:
```

Comment:
Fetching the "popComp" should be done after every successful connect and the cached. `get_pop_comp()` should return the cached value. This will help the (upcoming) re-connect logic.

Reply:
Got it. That was my initial approach but then it changes all the tests because we need to fake a get response now for every connect.